src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Dropdown from './Dropdown';
+import Button from '../Button';
+import DropdownGroupName from './DropdownGroupName';
+import Icon from '../Icon';
+
+const meta = {
```

Comment:
should we specify variant a la button to remove ghost since we'll be moving that into default for dark theme?